www/js/components/MetricsCard.tsx

```diff
@@ -0,0 +1,122 @@
+
+import React, { useRef, state, useState } from 'react';
+import { array, string, bool } from 'prop-types';
+import { angularize } from '../angular-react-helper';
+import { View } from 'react-native';
+import { useTheme , Card, IconButton, Surface, SurfaceProps} from 'react-native-paper';
+import BarChart from './BarChart';
+import MetricsDetails from './MetricDetails';
+import { StyleSheet } from 'react-native';
+
+
+const MetricsCard = ({chartData, axisTitle}) => {
+  
+  const { colors } = useTheme();
+  const [state, setState] = useState({detailsView : false})
+  const [ numVisibleDatasets, setNumVisibleDatasets ] = useState(1);
+
+  const defaultPalette = [
+    '#c95465', // red oklch(60% 0.15 14)
+    '#4a71b1', // blue oklch(55% 0.11 260)
+    '#d2824e', // orange oklch(68% 0.12 52)
+    '#856b5d', // brown oklch(55% 0.04 50)
+    '#59894f', // green oklch(58% 0.1 140)
+    '#e0cc55', // yellow oklch(84% 0.14 100)
+    '#b273ac', // purple oklch(64% 0.11 330)
+    '#f09da6', // pink oklch(78% 0.1 12)
+    '#b3aca8', // grey oklch(75% 0.01 55)
+    '#80afad', // teal oklch(72% 0.05 192) 
+  ]
+  return (
+    <Card children={{}} style={{width:"90%", alignSelf:"center", height:"280px" }}>
```

Comment:
You don't have to include `children` as a prop. I know VSCode probably yelled at you about it being missing.

As long as there are components nested inside you can ignore that warning. It's some issue with VSCode, Typescript, and React.